config/routes.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   authenticated :user do
-    root to: 'users#show', as: :dashboard
+    root to: redirect('/dashboard'), as: :authenticated_root
```

Comment:
Is `:authenticated_root` needed by Devise to work properly on a redirect? Couldn't see any explicit calls of `authenticated_root_path` etc

Reply:
Good question! thats a bit of a work around. Rails thinks the two root definitions are the same if one of them isn't named https://github.com/heartcombo/devise/issues/5084